app/Filament/Resources/Items/Schemas/ItemForm.php

```
@@ -19,36 +21,43 @@ public static function configure(Schema $schema): Schema
+                    ->label(__('Description')),
                 TextInput::make('quantity')
                     ->integer()
-                    ->required()
-                    ->minValue(0)
                     ->default(0)
                     ->readOnly()
-                    ->helperText(__('Auto-calculated from batches'))
-                    ->label(__('Quantity')),
+                    ->helperText(__('The quantity is computed from all batches'))
+                    ->label(__('Quantity'))
+                    ->hidden(static fn ($get, $record) => is_null($record)),
```

PHPMD (2.15.0)

Avoid unused parameters such as '$get'. (undefined)

(UnusedFormalParameter)